src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin_fixpt.c

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+
+/* Include Files */
+#include "drc_asin_fixpt.h"
+
+/* Function Declarations */
+static int32_T asr_s32(int32_T u, uint32_T n);
+static int8_T scalar_cordic_asin(int8_T sinValue, int16_T nIters, const int8_T
+  LUT[6]);
+
+/* Function Definitions */
+
+ /*
+ *-----------------------------------------------------------------------------
+ *
+ * Prototype       :static scalar_cordic_asin
+ * Description     :Compute the inverse Sine using a CORDIC implementation.
+ * ASR 32          : Arithmetic shift right  for 32 bit
+ * Arithmetic bit right shift is performed for CORDIC iteration
+ * For angles less than pi/4, we initialize (x,y) along the x-axis [ see code variable - xShift,xDShift].
```

Comment:
This feels like the overview comment, not for asr_s32?
If so, maybe put to the beginning of this file? And IMO this is a little bit too detailed.

Reply:
This feels like the overview comment, not for asr_s32?
 I make change for  Description section,